engine/class_modules/monk/sc_monk.cpp

```diff
@@ -327,16 +344,159 @@ struct monk_action_t : public Base
       p()->cooldown.bonedust_brew->adjust( timespan_t::from_seconds( time_reduction ), true );
   }
 
+   int get_bdb_zone()
+  {
+    std::vector<player_t*> target_list = p()->sim->target_non_sleeping_list.data();
+
+    int N           = static_cast<int>( target_list.size() );
```

Comment:
You seem to have very inconsistent format for local variables. Why not stick to snake_case?